modules/cnf-programming-the-sriov-fec-operator-acc100.adoc

```diff
@@ -0,0 +1,345 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="configuring-the-sr-iov-fec-operator-intel-vran-dedicated-accelerator-acc100_{context}"]
```

Comment:
Please match the ID to the file name, according to team guidelines.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata